client/boot.js.erb

```diff
@@ -70,11 +70,11 @@ angular.module('probedock', [
   'probedock.testSuiteSizeWidget',
   'probedock.userDetailsWidget',
   // components
+  'probedock.dataLabel',
```

Comment:
All other components declare one directive (I think). So this one should be named `dataLabels` to indicate that it declares multiple directives.